distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -566,6 +567,8 @@ type Decimal
        If the argument is `NaN` or `+/-Inf`, an `Arithmetic_Error` error is
        thrown.
 
+       Algorithm taken from https://stackoverflow.com/a/7211688.
```

Comment:
We probably should have some syntax for URLs in docs.

Meanwhile, at least on GH, the dot gets to be part of the URL and makes it 404: [https://stackoverflow.com/a/7211688.](https://stackoverflow.com/a/7211688.)

Reply:
Done.